### PR TITLE
Prettier upgrade and e2e type checks

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**e2e**
- enable more type checking rules for e2e tests
- add missing types for e2e tests
- add e2e folder for Prettier checks and fix formatting

**Prettier**
- upgrade Prettier to 2.x branch
- update print width to 150
- fix formatting issues

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
-  "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": "none",
+  "printWidth": 120
```
Comment (eromano) [inline .prettierrc:4]: I know that is just a personal preference but i think that with our modern monitor 150 or even  more is a better settings. I

Comment (DenysVuika) [reply .prettierrc:4]: No, it's not a personal preference. I've started with 180 like we have in other repo, but the code goes offscreen and one needs to scroll horizontally to see everything. I tried the following setup: VS Code with the file browser panel open, on the macBook. 120 was the most comfortable but I can check 150 too.

Comment (eromano) [reply .prettierrc:4]: if you could check it yes it will be nice :)

Comment (DenysVuika) [reply .prettierrc]: Updated, there are only a couple cases in e2e where one word goes off screen, the rest of the code is fine for 150 print width

**Pull request merged**